test/e2e/serial/tests/workload_placement_resources.go

```diff
@@ -0,0 +1,213 @@
+ * considering exactly one specific resources as discriminant. Other tests in the suite do similar
+ * things, but their focus is usually different. We acknowledge and expect some degree of overlap.
+ *
+ * In this set we want to explore all the resources
+ * and make sure the alignment is done correctly for each of them individually. For example:
+ * can the scheduler align correctly when all the resources but the memory are available?
+ * rinse and repeat for CPUs, devices, hugepages...
+ */
+
+var _ = Describe("[serial][disruptive][scheduler][byres] numaresources workload placement considering specific resources requests", func() {
+	var fxt *e2efixture.Fixture
+	var nrtList nrtv1alpha1.NodeResourceTopologyList
+
+	BeforeEach(func() {
+		Expect(serialconfig.Config).ToNot(BeNil())
+		Expect(serialconfig.Config.Ready()).To(BeTrue(), "NUMA fixture initialization failed")
+
+		var err error
+		fxt, err = e2efixture.Setup("e2e-test-workload-placement-resources")
+		Expect(err).ToNot(HaveOccurred(), "unable to setup test fixture")
+
+		err = fxt.Client.List(context.TODO(), &nrtList)
+		Expect(err).ToNot(HaveOccurred())
+
+		// Note that this test, being part of "serial", expects NO OTHER POD being scheduled
+		// in between, so we consider this information current and valid when the It()s run.
+	})
+
+	AfterEach(func() {
+		err := e2efixture.Teardown(fxt)
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	// note we hardcode the values we need here and when we pad node.
+	// This is ugly, but automatically computing the values is not straightforward
+	// and will we want to start lean and mean.
+
+	Context("with at least two nodes suitable", func() {
+		// FIXME: this is a slight abuse of DescribeTable, but we need to run
+		// the same code which a different test_id per tmscope
+		DescribeTable("[tier1][ressched] a guaranteed pod with one container should be placed and aligned on the node",
+			func(tmPolicy nrtv1alpha1.TopologyManagerPolicy, requiredRes, expectedFreeRes corev1.ResourceList) {
+				nrtCandidates, targetNodeName := setupNodes(fxt, desiredNodesState{
+					NRTList:           nrtList,
+					RequiredNodes:     2,
+					RequiredNUMAZones: 2,
+					RequiredResources: requiredRes,
+					FreeResources:     expectedFreeRes,
+				})
+
+				nrts := e2enrt.FilterTopologyManagerPolicy(nrtCandidates, tmPolicy)
+				if len(nrts) != len(nrtCandidates) {
+					Skip(fmt.Sprintf("not enough nodes with policy %q - found %d", string(tmPolicy), len(nrts)))
+				}
+
+				By("Scheduling the testing pod")
+				pod := objects.NewTestPodPause(fxt.Namespace.Name, "testpod")
+				pod.Spec.SchedulerName = serialconfig.Config.SchedulerName
+				pod.Spec.Containers[0].Resources.Limits = requiredRes
+
+				err := fxt.Client.Create(context.TODO(), pod)
+				Expect(err).NotTo(HaveOccurred(), "unable to create pod %q", pod.Name)
+
+				By("waiting for node to be up and running")
```

Comment:
nit: waiting for **pod** to be up and running

Reply:
thanks, fixed